### DISCUSSION TITLE
Is it possible to change background color of a button depending on state?

### DISCUSSION THREAD

**Discussion by canedje** [matt8707/ha-fusion#363] [Q&A]
Hello I'm learning about templating and learning a lot :-)
Is it possible to change the button background to a color depending on the state?
For example:
The device_tracker button is white when home and grey when not home.
Now I like to make the color white if state is "Horizon Pa" and grey if not?
![image](https://github.com/matt8707/ha-fusion/assets/23150947/41f743b5-2280-41de-a55b-38e05613f3b4)

**Comment by matt8707** [comment]
That is not possible

**Comment by canedje** [comment]
Thanks for the answer
